README.md

```diff
@@ -1 +1 @@
-# Valence Protocol
+# Valence Programs
```

Comment:
protocol -> programs as well?

Reply:
Yes, the repo will also be renamed eventually